wunderio/next-drupal-starterkit
# NEX-154: Prevent the Drupal backend from being indexed by search engines

We don't want the backend Drupal environment to be indexed by search engines, so in this PR we add the `noindex` metatag to all pages



To test: 

2. check that drupal pages have the `noindex` meta tag:

![image](https://github.com/wunderio/next-drupal-starterkit/assets/185412/e26ae11e-8733-4a9b-942f-5630c9897fd6)

But check that frontend pages do not have the noindex tag
